src/org/zaproxy/zap/extension/pscanrules/TestInfoPrivateAddressDisclosure.java

```diff
@@ -1,153 +1,159 @@
-/*

```

Comment:
Changed back, but still the same here - and my local Eclipse & Notebook++ - diffs show still just mark the really edited lines.